dls-controls/malcolmjs
# Feature/new table #212

## Description

Improved styling and added unit tests for new table

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/PANDA:SEQ1/table
- view the new table!

## Agile board tracking

connect to #212 
closes #212
